buf.ts.gen.yaml

```diff
@@ -2,5 +2,18 @@ version: v1
 managed:
```

Comment:
what's the relationship between `pnpm install -g protoc-gen-ts` and this file

Reply:
Yeah that's right.